dotnet/src/Connectors/Connectors.AI.OpenAI/AzureSdk/OpenAIChatRequestSettings.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using Microsoft.SemanticKernel.AI.ChatCompletion;
+
+namespace Microsoft.SemanticKernel.Connectors.AI.OpenAI.AzureSdk;
+
+// TODO: remove this temporary class
```

Comment:
This class will be removed once #2829 comes in. I just needed a request settings object in the connector to not mess up the whole dependency tree while testing.